modules/ROOT/pages/apply-rate-limiting-mule3.adoc

```diff
@@ -0,0 +1,20 @@
+= Rate Limiting Policy v1.0.0 or v1.1.0
```

Comment:
Mule 3.x does has not the policy versioning feature.

Reply:
Removed: https://github.com/mulesoft/docs-api-manager/pull/20/commits/c702571903784472e65171e288ed434e6b419a2e